SSSwiftUIAnimations/Sources/WaterProgressAnimation/WaterCircleView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  WaterCircleView.swift
+//  SSSwiftUIAnimations
+//
+//  Created by Rahul Yadav on 09/07/24.
+//
+
+import SwiftUI
+
+struct WaterCircleView: Shape {
+    
+    // MARK: - Variables
+    var progress: Double
+    
+    //Initial Animation Start
+    var offset: Angle
+    
+    //Enabling Animation
+    var animatableData: Double {
+        get { offset.degrees }
+        set { offset = Angle(degrees: newValue) }
+    }
+    
```

Comment:
Add function description and make internal, private, or file private function whichever is applicable. 
**Note**: check all the functions whichever you have created in other files as well